### PR TITLE
#1646 Add file size in streamApp page

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1646 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
The failing test will be address #1650

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/StreamApp/StreamApp.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ const StreamApp = props => {
   const rows = jars.map(jar => {
     return createData(
       jar.name,
-      999,
+      jar.size,
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/StreamApp/StreamApp.js]: Should this be subtracted by 1024? So it's actually displayed in KB?

**Comment by konekoya** [comment]
retry manager-it

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/StreamApp/StreamApp.js (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/StreamApp/StreamApp.js]: oh~
I forget should division 1024

**Comment by konekoya** [comment]
@wu87988622 Could you rebase this PR and fix the conflicts shown above? After that, I think it's ready to merge it. The failed e2e test will be addressed in #1635

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Overview/OverviewStreamApps.js [hunk 2]
```diff
@@ -50,8 +51,8 @@ const OverviewStreamApps = props => {
             return (
               <TableRow key={jar.name}>
                 <StyledTableCell>{jar.name}</StyledTableCell>
-                <StyledTableCell align="right">
-                  {parseInt(jar.size / 1024)}
+                <StyledTableCell align="right" >
+                {floor(divide(jar.size,1024),2)}
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Overview/OverviewStreamApps.js]: Is this a better way of achieving the same result?

File: ohara-manager/client/cypress/integration/workspacesPage.test.js [hunk 3]
```diff
@@ -120,7 +120,7 @@ describe('WorkspacesPage', () => {
       .get('@newTopic')
       .then(topic => {
         cy.getByTestId(topic.name)
-          .click()
+          .click({ force: true })
```
Comment (konekoya) [inline ohara-manager/client/cypress/integration/workspacesPage.test.js]: This won't fix the issue. See #1635 for the fix

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Overview/OverviewStreamApps.js (hunk 2 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Overview/OverviewStreamApps.js]: floor can computes number rounded down to precision.

**Comment by wu87988622** [comment]
retry manager

**Comment by konekoya** [comment]
The failed e2e test will be addressed in #1635

**Pull request merged**